exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/explain/NumberingRelWriter.java

```diff
@@ -125,14 +131,56 @@ protected void explain_(
     }
     pw.println(s);
     spacer.add(2);
-    explainInputs(inputs);
+    explainInputs(rel);
     spacer.subtract(2);
   }
 
-  private void explainInputs(List<RelNode> inputs) {
-    for (RelNode input : inputs) {
-      input.explain(this);
+  private String getDependentSrcOp(RelNode rel) {
+    if (rel instanceof UnnestPrel) {
+      return this.getDependentSrcOp((UnnestPrel) rel);
     }
+    return "";
+  }
+
+  private String getDependentSrcOp(UnnestPrel unnest) {
+    Prel parent = this.getRegisteredPrel(unnest.getParentClass());
+    if (parent != null && parent instanceof CorrelatePrel) {
+      OpId id = ids.get(parent);
+      return String.format(" [SrcOp: (%02d-%02d)] ", id.fragmentId, id.opId);
```

Comment:
Can you change this to 
```java
 return String.format(" [srcOp=%02d-%02d] ", id.fragmentId, id.opId);
```
It makes it easier to use regex and extract the operator Id with JavaScript

Reply:
Without the change, the visual plan rendering remains unchanged:
![image](https://user-images.githubusercontent.com/4335237/41141843-596c3fe4-6aa8-11e8-9e81-45ee5d9da144.png)


With the change, the visual plan is rndered:
![image](https://user-images.githubusercontent.com/4335237/41141833-4c848a84-6aa8-11e8-894b-b7d6be9a2274.png)